tests/foreman/ui/test_user.py

```diff
@@ -23,7 +23,7 @@
 from robottelo.ui.session import Session
 
 
-def valid_strings(len1=100):
+def valid_strings(len1=10):
```

Comment:
Got it. As this is a system behaviour then we should look for the rules to concatenate and then write some tests for them.